frontend/src/reducers/qnHistoryReducer.ts

```diff
@@ -178,13 +151,22 @@ export const updateQnHistoryById = async (
   >,
   dispatch: Dispatch<QnHistoryActions>
 ): Promise<boolean> => {
+  const accessToken = localStorage.getItem("token");
```

Comment:
Ah also I merged Nicole's PR earlier, so might want to update any token related portions according to her changes~